src/charts/bar.js

```diff
@@ -597,6 +589,34 @@ define(function(require) {
                     .attr('y2', chartHeight);
         }
 
+        /**
+         * Custom OnMouseOver event handler
+         * @return {void}
+         * @private
+         */
+        function customOnMouseOver(e, d, chartWidth, chartHeight) {
```

Comment:
Could we rename these to 'handleMouseOver', 'handleMouseMove' and 'handleMouseOut' ?